bentoml/adapters/utils.py

```diff
@@ -1,7 +1,8 @@
 import gzip
 import json
 import os
-from dependency_injector.wiring import Provide, inject
+
+from simple_di import Provide, inject
```

Comment:
Should we maintain `simple_di` as a separate project? Here are some questions to consider. 

- Will we have other future projects other than BentoML using this library?
- Will we actively support open source requests?

Reply:
1. > Will we have other future projects other than BentoML using this library?
    I'd like to maintain/test it independently because its logic is standalone and consolidate.
2. IMO there won't be many requests in the future tho. If there was, it would't be a burden since it's a very small lib.